### PR TITLE
fix for #140, #141

### PR DESCRIPTION
**Pull request by TaskSquirrel**
Fixes for #140 and #141 
Plus some styling changes to match spec
* transparent background on inactive filter buttons on merchants page
* flyer message left aligned

Before:
image

After:
image

### CODE REVIEW

**Comment by danthaman44** [comment]
Can you upload a screenshot of the UI after these changes?

File: src/components/MerchantsPage/styles.module.scss [hunk 1]
```diff
@@ -145,8 +145,17 @@
 }
 
 .flyerContainer {
-  margin: 3vh 7vw;
+  $flyer-margin: 3rem;
+
+  margin-top: $flyer-margin;
+  margin-bottom: $flyer-margin;
+  font-size: 120%;
```
Comment (jtmckibb) [inline src/components/MerchantsPage/styles.module.scss:152]: Why is the font size a percent?

File: src/index.scss [hunk 2]
```diff
@@ -107,6 +107,7 @@ h3 {
 
 .modalButton {
   &--back {
+    background: none;
```
Comment (jtmckibb) [inline src/index.scss:110]: What does this fix?

Comment (TaskSquirrel) [reply src/index.scss:110]: Removes background color from the filtering buttons on the top of the screenshots in Chrome; appears to be using the Chrome default button bg color

**Comment by TaskSquirrel** [comment]
Added before & after (top filtering buttons and bottom flyer message)

File: src/components/MerchantsPage/styles.module.scss (hunk 1 above)
Comment (TaskSquirrel) [reply src/components/MerchantsPage/styles.module.scss:152]: The text in the flyer msg seems to be slightly larger than the normal text; is there any text component/mechanism you guys are using to keep font sizes consistent?

**Pull request merged**